src/foundry/foundry.js/applications/pause.d.ts

```diff
@@ -1,13 +1,13 @@
 /**
  * Pause notification in the HUD
+ *
+ * @typeParam Options - the type of the options object
  */
-declare class Pause<D extends Pause.Data = Pause.Data> extends Application {
```

Comment:
Why remove the ability to extend it? What if somebody wants to create a custom `Pause` that actually needs more / other properties in the handlebars template?

Reply:
I guess I followed the schema from `Hotbar`. But you are right, it is better to be flexible here.